edx_adapt/tests/logic_tests.py

```diff
@@ -36,17 +36,20 @@ def tearDownClass(cls):
         regex = re.compile('_student_')
         mclient.Generic.remove({'key': {'$regex': regex}})
 
-    def _answer_pre_assessment_problems(self, correct=0, attention_question=True):
+    def _answer_pre_assessment_problems(self, correct_answers=0, attention_question=True):
         """
         Automation fulfilling Pre-Assessment problems
 
-        :param correct: int, Number of correct answers
+        :param correct_answers: int, Number of correct answers
         :param attention_question: bool, by default one of correct answers id assigned to AttentionQuestion
         """
         pre_assessments = ['Pre_assessment_{}'.format(i) for i in range(0, 14)]
-        for i, problem in enumerate(pre_assessments):
-            correct_value = True if i < correct - attention_question or (attention_question and i == 13) else False
-            data = {'problem': problem, 'correct': correct_value, 'attempt': 1}
+        for answered_problem, problem in enumerate(pre_assessments):
+            is_correct = (
+                answered_problem < correct_answers - attention_question or
```

Comment:
I still don't like the implicit type conversion and a magic number . Is there a better way to express the correctness check?